www/src/libs/math.js

```diff
@@ -317,6 +317,7 @@ var _mod = {
        var x1=float_check(x);
        var y1=float_check(y);
        return float(Math.sqrt(x1*x1 + y1*y1))},
+    inf: float(Infinity),
```

Comment:
Yes, and it should be changed in several lines of code across all brython.

* We could define this as a new milestone for next versions, 
* we could stick just with newer versions of browsers (this is a nightmare) and let it as it is right now.

In general, working with numbers in JS is a real pain and a lot of checks have to be made on the Brython side making the code slower, memory consuming and heavier.

@PierreQuentel what do you think about supporting older browsers than that supporting ES5 (2011)?
@PierreQuentel what do you think about changing all the references to `Infinity`? I can be in charge of this.